package.json

```diff
@@ -61,20 +61,21 @@
     ]
   },
   "devDependencies": {
+    "arg": "5.0.0",
     "husky": "^1.3.1",
     "lerna": "^3.22.1",
     "lint-staged": "^8.1.0",
     "prettier": "^2.1.2",
-    "prettier-config-carbon": "^0.4.0",
-    "arg": "5.0.0"
+    "prettier-config-carbon": "^0.4.0"
   },
   "sideEffects": false,
   "version": "0.0.0-development",
   "resolutions": {
-    "carbon-addons-iot-react/chokidar": "3.3.1",
```

Comment:
I think is the package resolution syntax we were discussing in scrum. Is there a reason we are removing the scope of our package?

Reply:
That's a good call. I was playing around figuring out how those worked exactly. I'll add it back in.

Reply:
re-added.